pkg/collector/corechecks/gpu/gpu.go

```diff
@@ -0,0 +1,203 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux_bpf
```

Comment:
I don't think the `linux_bpf` build tag is used for the core agent, only system-probe.